src/In.ProjectEKA.HipService/DataFlow/DataFlowConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+namespace In.ProjectEKA.HipService.DataFlow
+{
+    public class DataFlowConfiguration
+    {
+        public int DataSizeLimitInMbs { get; set; }
+        public int DataLinkTTLInMinutes { get; set; }
+
+        public DataFlowConfiguration()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5067092)